src/Worker/DBClean.php

```diff
@@ -22,6 +22,7 @@
 namespace Friendica\Worker;
 
 use Friendica\Core\Logger;
+use Friendica\Core\System;
```

Comment:
Where does this dependency come from? I don't see any added usage of it

Reply:
You're right, I removed it.